sources/packages/api/src/route-controllers/student-account-applications/student-account-application.aest.controller.ts

```diff
@@ -0,0 +1,172 @@
+import {
+  BadRequestException,
+  Body,
+  Controller,
+  Get,
+  Injectable,
+  NotFoundException,
+  Param,
+  ParseIntPipe,
+  Patch,
+  Post,
+} from "@nestjs/common";
+import { ApiNotFoundResponse, ApiTags } from "@nestjs/swagger";
+import { ClientTypeBaseRoute } from "../../types";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import {
+  AllowAuthorizedParty,
+  Groups,
+  Roles,
+  UserToken,
+} from "../../auth/decorators";
+import { UserGroups } from "../../auth/user-groups.enum";
+import BaseController from "../BaseController";
+import {
+  FormNames,
+  FormService,
+  StudentAccountApplicationsService,
+} from "../../services";
+import {
+  StudentAccountApplicationAPIOutDTO,
+  StudentAccountApplicationApprovalAPIInDTO,
+  StudentAccountApplicationSummaryAPIOutDTO,
+} from "./models/student-account-application.dto";
+import { CustomNamedError, getUserFullName } from "../../utilities";
+import { IUserToken } from "../../auth/userToken.interface";
+import { STUDENT_ACCOUNT_APPLICATION_NOT_FOUND } from "../../constants";
+import { Role } from "../../auth/roles.enum";
+
+/**
+ * Student account applications when the authentication happens through BCeID
+ * and the Ministry needs to confirm the student identity.
+ */
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("student-account-application")
+@ApiTags(`${ClientTypeBaseRoute.AEST}-student-account-application`)
+@Injectable()
+export class StudentAccountApplicationAESTController extends BaseController {
+  constructor(
+    private readonly formService: FormService,
+    private readonly studentAccountApplicationsService: StudentAccountApplicationsService,
+  ) {
+    super();
+  }
+
+  /**
+   * Get the summary list of all student account applications
+   * waiting to be assessed by the Ministry.
+   * @returns summary list of pending student account applications.
+   */
+  @Get("pending")
+  async getPendingStudentAccountApplications(): Promise<
+    StudentAccountApplicationSummaryAPIOutDTO[]
+  > {
+    const accountApplications =
+      await this.studentAccountApplicationsService.getPendingStudentAccountApplications();
+    return accountApplications.map((accountApplication) => ({
+      id: accountApplication.id,
+      fullName: getUserFullName(accountApplication.user),
+      submittedDate: accountApplication.submittedDate,
+    }));
+  }
+
+  /**
+   * Get the student account application previously submitted
+   * by the student for the basic BCeID student account creation.
+   * @param id student account application id.
```

Comment:
param name is `studentAccountApplicationId`

Reply:
Added here and to others also.